sdcm/nemesis.py

```diff
@@ -136,80 +152,12 @@
 )
 
 
-class NoFilesFoundToDestroy(Exception):
-    pass
-
-
-class NoKeyspaceFound(Exception):
-    pass
-
-
-class FilesNotCorrupted(Exception):
-    pass
-
-
-class LogContentNotFound(Exception):
-    pass
-
-
-class LdapNotRunning(Exception):
-    pass
-
-
-class TimestampNotFound(Exception):
-    pass
-
-
-class PartitionNotFound(Exception):
-    pass
-
-
-class TriggerCallableException(Exception):
-    """ raised when a trigger function in a trigger - watcher pair fails"""
-
-
-class WatcherCallableException(Exception):
-    """ raised when a watcher function in a trigger - watcher pair fails"""
-
-
-class UnsupportedNemesis(Exception):
-    """ raised from within a nemesis execution to skip this nemesis"""
-
-
-class NoMandatoryParameter(Exception):
-    """ raised from within a nemesis execution to skip this nemesis"""
-
-
 class DefaultValue:  # pylint: disable=too-few-public-methods
     """
     This is class is intended to be used as default value for the cases when None is not applicable
     """
     ...
 
```

Comment:
Same here, one more empty line must be added.

Reply:
fixed